packages/@storefront/core/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [patch]
+### Added
+- SF-1241: Expose table of DOM key strings.
+  - This table may be used by StoreFront components and/or implementations which must respond to DOM events.
```

Comment:
Can you list the KEYS table here as well?

Reply:
Noted, will do.